PrebidMobile/PrebidMobile-core/src/test/java/org/prebid/mobile/rendering/models/openrtb/bidRequests/PluginRendererListTest.java

```diff
@@ -7,10 +7,18 @@
 import org.junit.Test;
 
 import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Iterator;
 import java.util.List;
 
 public class PluginRendererListTest {
 
+    HashMap<String, Object> data = new HashMap<String, Object>() {{
+        put("extra1", true);
+        put("extra2", 1);
+        put("extra3", "value");
```

Comment:
can you add a nested HasMap object as a value inside the current data HashMap
resulting in a JSON like this:
```
{
  "extra4": {
    "key1": "value1"
    "key2": true
  }
}
```